projects/event-lambdas/src/event-api-lambda/application.ts

```diff
@@ -65,5 +68,53 @@ export const createApp = (initConfig: AppConfig): express.Application => {
       )
   );
 
+    app.get(
+        "/tracking-pixel",
+        async (req: Request, res: Response) =>
+            authenticatePandaUser(
+                initConfig.panDomainAuthentication,
+                req,
+                res,
+                async ({ email }: User) => {
+                    const {app, stage} = req.query;
+                    const {hostname, pathname} = url.parse(req.header("referrer") || "");
+
+                    if(!email ||
+                        !app || typeof app !== "string" ||
+                        !stage || typeof stage !== "string"
+                    ) {
+                        applyErrorResponse(
+                            res,
+                            400,
+                            "Incorrect event format"
+                        );
+                        return;
+                    }
+
+                    const toolView: IUserTelemetryEvent = {
+                        app: "tools-audit",
+                        stage: "INFRA",
+                        type: "GUARDIAN_TOOL_ACCESSED",
```

Comment:
Events are stored in the format: `[app]/[stage]/[type]`.

Setting the app to `tools-audit` will help us keep these records grouped together. The alternative would be to group them by the tool they are tracking, but that would split them into different directories.

Reply:
We had an offline discussion about this. There are a few considerations:

- Making PII easy to locate and if necessary delete
- How to best co-locate data in the case of shifting 
- Current directory convention
- Sanitising user inputs

The intention to put all tracking under a single App was to address the first point. 

As an alternative, @jonathonherbert  suggested we could add a root `data-pii` directory, with the rest of the convention remaining as is i.e. `[app]/[stage]/[type]`.

This seems sensible, however, in the case of PII, it may be most valuable to group buckets by identifiers. This would make it easy to retrieve and erase data. 

We are still waiting to hear back from the DPO. Her input may help inform this decision.

Reply:
I have backed out the S3 persisting for now as we have ongoing conversations with our DPO. In the short term, we will log out events which will be available for 10 days in our logging platform.